archive/p/php/README.md

```diff
@@ -8,6 +8,7 @@ Welcome to Sample Programs in PHP!
 - [Fizz Buzz in PHP][3]
 - [Baklava in PHP][8]
   - Solution borrowed from @toturkmen and @erayak via the [baklava repo][1]
+- Fibonacci in PHP
```

Comment:
Can you add an article issue like #33 and link it here?

Reply:
Sure! #941